hledger, hledger-ui, hledger-web: update/add (#12824)
tldr-pages/tldr: pages/common/hledger.md

# hledger

> > A robust, friendly plain text accounting app (the command-line interface).

Reviewer: ```suggestion
> A robust, friendly plain text accounting app.
> See also: `hledger-ui` for TUI, `hledger-web` for web interface.
```

> More information: <https://hledger.org>.

- Record new transactions interactively, saving to the default journal file:

`hledger add`

> - Import new transactions from bank.csv, using bank.csv.rules to convert:

Author: I wanted to give the hint that reading foo.csv uses and requires a nearby foo.csv.file. "Use its rules" seems a little more obscure for new users ?

> - Import new transactions from bank.csv, using bank.csv.rules to convert:

Reviewer: > I wanted to give the hint that reading foo.csv uses and requires a nearby foo.csv.file. "Use its rules" seems a little more obscure for new users ?

Agreed, will revert that section of my suggestion.

`hledger import {{bank.csv}}`

- Print all transactions, reading from multiple specified journal files:

> `hledger print --file {{2024.journal}} --file {{2023.journal}}`

Reviewer: ```suggestion
`hledger print --file {{path/to/file1.journal}} --file {{path/to/file2.journal}}`
```

- Show all accounts, as a hierarchy, and their types:

`hledger accounts --tree --types`

- Show asset and liability account balances, including zeros, hierarchically:

`hledger balancesheet --empty --tree --no-elide`

- Show monthly incomes/expenses/totals, largest first, summarised to 2 levels:

`hledger incomestatement --monthly --row-total --average --sort --depth 2`

> - Show the assets:bank:checking account's transactions and running balance:

Reviewer: ```suggestion
- Show the `assets:bank:checking` account's transactions and running balance:
```

`hledger aregister assets:bank:checking`

> - Show the amount spent on food from the assets:cash account:

Reviewer: ```suggestion
- Show the amount spent on food from the `assets:cash` account:
```

Author: I wasn't sure if this content is considered markdown; I don't think it is. Eg `tldr` on mac just renders the backquotes as written. In that case, would single quotes or double quotes be preferable ? Backquotes have specific meaning to the shell and would cause problems if a user copies and pastes them.

Reviewer: In descriptions, when backticks are used, it is highlighted by our clients in a different style/color which doesn't only mean a command in this context but can also mean something like file names, properties, etc.

i.e. This is from the Git page we have:

![image](https://github.com/tldr-pages/tldr/assets/26346867/a6ef5728-02c0-4c88-82c6-dc2dd170a72e)

# hledger

> A robust, friendly plain text accounting app.
> See also: `hledger-ui` for TUI, `hledger-web` for web interface.
> More information: <https://hledger.org/hledger.html>.

- Record new transactions interactively, saving to the default journal file:

`hledger add`

> - Import new transactions from bank.csv, using bank.csv.rules to convert:

Reviewer: ```suggestion
- Import new transactions from `bank.csv`, using `bank.csv.rules` to convert:
```
